packages/ckeditor5-ui/src/dialog/dialog.ts

```diff
@@ -0,0 +1,143 @@
+/**
+ * @license Copyright (c) 2003-2023, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ */
+
+/**
+ * @module ui/dialog/dialog
+ */
+
+import View from '../view';
+import { type Editor, Plugin } from '@ckeditor/ckeditor5-core';
+import DialogView, { type DialogViewCloseEvent, DialogViewPosition } from './dialogview';
+import type { DialogActionButtonDefinition } from './dialogactionsview';
+
+import '../../theme/components/dialog/dialog.css';
+
+/**
+ * TODO
+ */
+export default class Dialog extends Plugin {
+	/**
+	 * TODO
+	 */
+	public readonly view: DialogView;
```

Comment:
We should either have `isOpen` property or the `view` should be `DialogView | null` type and it should be created/destroyed each time the dialog is opened/closed.

Reply:
For now, I've settled for `_isOpen` private prop. It may turn out useful to turn it into a public observable in the future, but it wasn't necessary yet, so I kept is simple.

Reply:
Private prop does not give us what I wanted to achieve - I want external code to be able to check if there's any dialog open.